kobo/apps/accounts/templates/account/signup.html

```diff
@@ -91,6 +91,5 @@ <h2>{% trans "Register using SSO" %}</h2>
   </div>
   <div style="clear:both;"></div>
   {% include "../legal/registration_legal.html" with config=config %}
-</div>
```

Comment:
An extra `</div>` [snuck](https://github.com/kobotoolbox/kpi/pull/5090/commits/649d68caf4f1b5135fde43d04b1c4936fa9b3a2c#diff-316fadeed80679fe9a99ac2e78a3b2079a1de730ae3f9cd243af15677ca79e20R105) in here ([I removed it](https://github.com/kobotoolbox/kpi/pull/5090/commits/0fd278b6616e09fa44f8354ce5161748ed0872ad))

Was causing the photo credit to do this:


| before | after |
| ------ | ----- |
| ![Screenshot_20241011_185218](https://github.com/user-attachments/assets/5b1ec3d7-18fd-43c2-8aa6-13f0220b52cd)| ![Screenshot_20241011_210602](https://github.com/user-attachments/assets/1478cfe6-2416-4cbc-9657-a501996c17f1) |